File: 69a.py
n  = int(input())
arr=[]
sum_ =int(0)
for i in range(n):
    a=[]
    b = input()
    lst=b.split()
    for j in range(3):
        a.append(lst[j])
        sum_=sum_+int(lst[j])
    arr.append(a)
        
if(sum_==0):
    print("YES")
else:
    print("NO")       

File: 69A.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    int sum1=0,sum2=0,sum3=0,sum=0;
    cin>>n;
    int arr[n][3];
    for(int i=0;i<n;i++){
        for(int j=0;j<3;j++){
            cin>>arr[i][j];
            sum=sum+arr[i][j];
        }
    }
    for(int i=0;i<n;i++){
       sum1=sum1+arr[i][0];
        
    }
    for(int i=0;i<n;i++){
       sum2=sum2+arr[i][1];
        
    }
    for(int i=0;i<n;i++){
       sum3=sum3+arr[i][2];
        
    }
    if(sum1==0&&sum2==0&&sum3==0&&sum==0)
    cout<<"YES"<<endl;
    else
    cout<<"NO"<<endl;
    return 0;
}